source/gloperate/source/base/Canvas.cpp

```diff
@@ -81,7 +81,7 @@ void Canvas::setRenderStage(std::unique_ptr<Stage> && stage)
 
 void Canvas::onRender(globjects::Framebuffer * targetFBO)
 {
-    cppassist::debug(1) << "onRender()";
+    cppassist::debug(2) << "onRender()";
```

Comment:
I think the target FBO should be subject to log, too.
I suggest adding this information at debug level ```+3``` and above.